src/pocketmine/Player.php

```diff
@@ -918,6 +918,27 @@ protected function checkItemCooldowns() : void{
 		}
 	}
 
+	/**
+	 * @return mixed
+	 */
+	public function getDeviceOS() {
+		return $this->deviceOS;
+	}
+
+	/**
+	 * @return mixed
+	 */
+	public function getDeviceModel() {
+		return $this->deviceModel;
+	}
+
+	/**
+	 * @return mixed
+	 */
+	public function getDeviceId() {
+		return $this->deviceId;
+	}
+
```

Comment:
You should remove the PHPDocs and add the correct type hints (not mixed)

Reply:
These methods are also in the wrong place, they are not near anything related to them.